tests/countries/test_bonaire_sint_eustatius_and_saba.py

```
@@ -0,0 +1,314 @@
+                    "2020-11-16",
+                    "2021-11-16",
+                    "2022-11-16",
+                    "2023-11-16",
+                    "2024-11-16",
+                    "2025-11-16",
+                )
+                self.assertHolidayName(name, holidays, range(2011, 2050))
+            else:
+                self.assertNoHolidayName(name, holidays)
+
+    def test_saba_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)